core/src/edu/asu/c3simulator/simulation/business/ProductionInformation.java

```diff
@@ -0,0 +1,60 @@
+package edu.asu.c3simulator.simulation.business;
+
+/**
+ * Structure to represent how, and in what quantities a {@link Product} is being produced.
+ * 
+ * @author Moore, Zachary
+ *
+ */
+public class ProductionInformation
+{
+	/** Product for which this information applies */
+	private Product target;
+	
+	/** Number of units of {@link #target} being produced by this company each month */
+	private int monthlyProduction;
+	
+	/** Amount spent on materials per unit produced */
+	private float materialBudgetPerUnit;
+	
```

Comment:
Shouldn't there be a constructor for this class? It will help ensure that a target is set for the information to belong to it; since there shouldn't be production information for no target.